docs/getstarted.md

```diff
@@ -176,49 +154,47 @@ The following figure illustrates this:
 
 ![Multiple clusters/VPCs service-to-service communications](images/example2.png)
 
-**Steps**
+### Steps
 
+**Set up `inventory-ver2` service and serviceExport on a second cluster** 
 
-   **Set up inventory on a second cluster** 
+1. Create a second Kubernetes cluster `cluster2` (using the same instructions used to create the first).
 
-1. Create a second cluster (using the same instructions used to create the first).
-   **Important**: Note that each cluster requires its own gateway.
-
-1. Ensure you're using the second cluster profile. 
+2. Ensure you're using the second cluster `kubectl` context. 
```

Comment:
putting 1 every lines is a fine way of numbering in markdown, you don't need to change that (it makes it harder to add a step in between)

Reply:
I think people who read the raw .md could be more clear on that is which step?

Reply:
The benefits are obvious especially for this kind of long list, which we will have more inserts and deletes. Markdown is a markup language designed to render, not really for raw text viewers.

I'm thinking of this as a source code of a docs website, so I would value maintainability more.